CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### New features
 
+* [#1712](https://github.com/bbatsov/rubocop/pull/1712): Set `Offense#corrected?` to `true`, `false`, or `nil` when it was, wasn't, or can't be auto-corrected, respectively. ([@vassilevsky][])
```

Comment:
Does this change manifest in an user visible manner? Looking at the code, I don't think so.